fuel-samples/weather/src/main/kotlin/fuel/samples/FuelWeather.kt

```diff
@@ -6,19 +6,41 @@ import fuel.Fuel
 import fuel.moshi.toMoshi
 import fuel.request
 import kotlinx.coroutines.runBlocking
+import kotlin.system.exitProcess
 
 @JsonClass(generateAdapter = true)
 data class Weather(
-    val title: String,
-    val latt_long: String
+        val title: String,
+        val latt_long: String,
+        val woeid: Int
+)
+
+@JsonClass(generateAdapter = true)
+data class ConsolidatedWeather(
+        val consolidated_weather: List<ConsolidatedWeatherEntry>
+)
+
+@JsonClass(generateAdapter = true)
+data class ConsolidatedWeatherEntry(
+        val applicable_date: String,
+        val weather_state_name: String,
+        val the_temp: Float
 )
 
 fun main() {
```

Comment:
This produces somewhat more meaningful command-line application :p

<img width="521" alt="Screen Shot 2021-02-22 at 15 40 11" src="https://user-images.githubusercontent.com/4669517/108671802-59280700-7524-11eb-9e80-cda8df77d580.png">